mix.exs

```diff
@@ -18,12 +18,13 @@ defmodule PhoenixPubsubRedis.Mixfile do
   end
 
   def application do
-    [applications: [:logger, :redo, :poolboy]]
+    [applications: [:logger, :poolboy, :redix, :redix_pubsub]]
   end
 
   defp deps do
-    [{:phoenix, "~> 1.1"},
-     {:redo, "~> 2.0.1"},
+    [{:phoenix_pubsub, "~> 1.0"},
+     {:redix, "~> 0.4.0"},
```

Comment:
We can remove this from dependencies since `redix_pubsub` depends on `redix` (also it starts application).